src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -259,7 +264,15 @@ def _normalize_properties(self, user, subscriptions, is_service_principal, cert_
                 _IS_DEFAULT_SUBSCRIPTION: False,
                 _TENANT_ID: s.tenant_id,
                 _ENVIRONMENT_NAME: self.cli_ctx.cloud.name
-            })
+            }
+            # for Subscriptions - List REST API 2019-06-01's subscription account
+            if subscription_dict[_SUBSCRIPTION_NAME] != _TENANT_LEVEL_ACCOUNT_NAME:
+                if hasattr(s, 'subscription_tenant_id'):
```

Comment:
what is the difference between subscription_tenant_id and tenant_id?

Reply:
Explained at https://github.com/Azure/azure-cli/pull/11886/files#diff-e35be24f5c5a220ff6948bcd6f18150cR37-R41

Reply:
Good point. I also think these concepts are very confusing. I am following up with Lighthouse and our PMs trying to persuade them to change `subscriptionTenantId` to `homeTenantId` which is a read-only field indicating where the subscription is hosted. `managedByTenants` is a mapping from REST API's [`managedByTenants`](https://docs.microsoft.com/en-us/rest/api/resources/subscriptions/list#subscription).